src/test/pool.test.ts

```diff
@@ -24,33 +24,38 @@ let dhedge: Dhedge;
 
 jest.setTimeout(100000);
 
-// const options = {
-//   gasLimit: 1000000,
-//   gasPrice: ethers.utils.parseUnits("35", "gwei")
-// };
+const options = {
+  gasLimit: 5000000,
```

Comment:
why is higher gas limit needed here?

Reply:
The tx needs about 1.5M and ethers.js  estimated too low. So the tx was erroring out when is set too low.